lib/notifications/client/helper_methods.rb

```diff
@@ -0,0 +1,13 @@
+require "base64"
+
+module Notifications
+  class Client
+    module HelperMethods
+      def prepare_upload(file)
+        raise ArgumentError.new("File size is over the 2 Mb limit") if file.size > MAX_FILE_UPLOAD_SIZE
+
+        { file: Base64.strict_encode64(file.read) }
```

Comment:
I'd add a `return` here. It doesn't make a difference for the users, but since most of us are more familiar with Python it could make this a bit easier to read for someone who hasn't done much ruby before (and implicit return is a hard thing to figure out if you haven't seen it before).

Reply:
I'm not keen on adding an explicit `return` here. Explicit returns go against our Ruby styleguide, so the `govuk-lint` gem gives an error. We also currently use implicit returns in a number of other places in the code (e.g. the integration tests).